lib/identity/finalize.rb

```diff
@@ -0,0 +1,30 @@
+module Identity
+  class Finalize < Default
+    register Sinatra::Namespace
+    include Helpers::Auth
+
+
+    namespace "/finalize" do
```

Comment:
I'm not super in love with this URL seeing as it doesn't really describe what this endpoint does. Did you guys consider any alternatives? I wonder if we should try something like `/login/external`, `/login/with-token`, `/login/with-authorization` or something?